bin/node/runtime/src/lib.rs

```diff
@@ -1972,7 +1972,21 @@ impl_runtime_apis! {
 
 	impl pallet_nomination_pools_runtime_api::NominationPoolsApi<Block, AccountId, Balance> for Runtime {
 		fn pending_rewards(member_account: AccountId) -> Balance {
-			NominationPools::pending_rewards(member_account).unwrap_or_default()
+			NominationPools::api_pending_rewards(member_account).unwrap_or_default()
```

Comment:
maybe `who` instead of `member_account`, as any AccountD can be passed to this function and the function can handle accounts that are not members.